primitives/src/v1.rs

```diff
@@ -60,33 +60,50 @@ pub const INCLUSION_INHERENT_IDENTIFIER: InherentIdentifier = *b"inclusn0";
 pub fn collator_signature_payload<H: AsRef<[u8]>>(
 	relay_parent: &H,
 	para_id: &Id,
+	validation_data_hash: &Hash,
 	pov_hash: &Hash,
-) -> [u8; 68] {
+) -> [u8; 100] {
```

Comment:
Could we consider adding a wrapper type which then exposes `Eq + PartialEq` as well as `as_slice()`? In this case it might be overkill, since the `fn` is used once afaics.

Reply:
I think it's overkill, especially since slices themselves are `Eq + PartialEq`.